src/tailwind.css

```
@@ -31,6 +31,14 @@
   }
 }
 
+@utility modal-overlay {
```

Stylelint (17.11.0)

[error] Unexpected unknown at-rule "@utility" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)